dev/doc/release-process.md

```diff
@@ -126,9 +126,9 @@ in time.
 - [ ] Send an e-mail on Coqdev announcing that the tag has been put so that
   package managers can start preparing package updates (including a
   `coq-bignums` compatible version).
-- [ ] Ping **@erikmd** to update the Docker images in `coqorg/coq`
-  (requires `coq-bignums` in `extra-dev` for a beta / in `released`
-  for a final release).
+- [ ] When opening the corresponding PR in the opam repository ([`coq/opam-coq-archive`](https://github.com/coq/opam-coq-archive) or [`ocaml/opam-repository`](https://github.com/ocaml/opam-repository)),
+  the package managers can Cc **@erikmd** to anticipate the Docker release in [`coqorg/coq`](https://hub.docker.com/r/coqorg/coq)
+  (need for a `coq-bignums` opam package in [`extra-dev`](https://github.com/coq/opam-coq-archive/tree/master/extra-dev), resp. [`released`](https://github.com/coq/opam-coq-archive/tree/master/released)).
```

Comment:
I don't understand the parenthetical comment.

Would be nice to spell out "resp.", do you mean "respectively"?

Reply:
@jfehrle thanks for your proofreading, indeed the grammar of the parenthetical comment wasn't very clear, I'm going to push a new version.